cloudmarker/test/test_gcpcloud.py

```diff
@@ -112,3 +112,287 @@ def test_instance_with_next_page_token(
         list(gcpcloud.GCPCloud('', '').read())
 
         self.assertEqual(mock_execute.mock_calls, [mock.call(), mock.call()])
+
+    def _patch(self, target):
+        patcher = mock.patch('cloudmarker.clouds.gcpcloud.' + target)
+        self.addCleanup(patcher.stop)
+        return patcher.start()
+
+    def setUp(self):
```

Comment:
I really like this approach to patching the test_methods. I was completely unaware of this. Thanks for introducing this to me :)

Reply:
Yes, this helps in reducing the boilerplate required for patching the same objects for every test method and specifying the same mock arguments for every test method.

By the way, patching the same objects for every test method can be avoided by specifying the patch decorator on top of the entire test class. But we still need to specify the same mock arguments for every test method.

This approach takes care of avoiding both types of boilerplate. It's good to know that you like it.